repos/system_upgrade/common/libraries/distro.py

```diff
@@ -16,3 +18,56 @@ def get_distribution_data(distribution):
         raise StopActorExecutionError(
             'Cannot find distribution signature configuration.',
             details={'Problem': 'Distribution {} was not found in {}.'.format(distribution, distributions_path)})
+
+
+def get_distro_repofiles(distro=None):
+    """
+    Get distribution provided repofiles
+
+    Note that this does not perform any validation, the caller must check
+    whether the files exist.
+
+    :return: A list of paths to repofiles provided by distribution
+    :rtype: List(String)
+    """
+    distro = distro or get_distro_id()
+    distro_provided_repofiles = {
+        "rhel": [
```

Comment:
mixture of quotes and apostrophes in the file. use please just one of them. preferably apostrophes if these are used already originally in the file.